File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-05-15/examples-java/CosmosDBTableGet.java
import com.azure.core.util.Context;

/** Samples for TableResources GetTable. */
public final class Main {
    /*
     * x-ms-original-file: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-05-15/examples/CosmosDBTableGet.json
     */
    /**
     * Sample code: CosmosDBTableGet.
     *
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void cosmosDBTableGet(com.azure.resourcemanager.AzureResourceManager azure) {
        azure
            .cosmosDBAccounts()
            .manager()
            .serviceClient()
            .getTableResources()
            .getTableWithResponse("rg1", "ddb1", "tableName", Context.NONE);
    }
}

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-08-15-preview/examples-js/CosmosDBTableGet.js
const { CosmosDBManagementClient } = require("@azure/arm-cosmosdb");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Gets the Tables under an existing Azure Cosmos DB database account with the provided name.
 *
 * @summary Gets the Tables under an existing Azure Cosmos DB database account with the provided name.
 * x-ms-original-file: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-08-15-preview/examples/CosmosDBTableGet.json
 */
async function cosmosDbTableGet() {
  const subscriptionId = "subid";
  const resourceGroupName = "rg1";
  const accountName = "ddb1";
  const tableName = "tableName";
  const credential = new DefaultAzureCredential();
  const client = new CosmosDBManagementClient(credential, subscriptionId);
  const result = await client.tableResources.getTable(resourceGroupName, accountName, tableName);
  console.log(result);
}

cosmosDbTableGet().catch(console.error);
